lib/node_modules/@stdlib/math/base/special/minmax/src/main.c

```diff
@@ -0,0 +1,60 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <stdlib.h>
+#include "stdlib/math/base/special/minmax.h"
+#include "stdlib/math/base/assert/is_nan.h"
+
+/**
+* Evaluates the min and max of two values.
+*
+* @param x   First value
+* @param y   Second value
+* @param min   to store minimum value 
+* @param max   to store maximum value
+*
+* @example
+* void y = stdlib_base_minmax( NaN, -1.0, &min, &max );
+* // min = NaN, max = NaN
+*
+* @example
+* void y = stdlib_base_minmax( 4.0, -5.0, &min, &max );
+* // min = -5.0, max = 4.0
+*/
```

Comment:
We can follow https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/rempio2/src/main.c#L450 here.